tfe/resource_tfe_team_members.go

```diff
@@ -92,9 +92,9 @@ func resourceTFETeamMembersUpdate(d *schema.ResourceData, meta interface{}) erro
 	tfeClient := meta.(*tfe.Client)
 
 	if d.HasChange("usernames") {
-		old, new := d.GetChange("usernames")
-		oldUsers := old.(*schema.Set).Difference(new.(*schema.Set))
-		newUsers := new.(*schema.Set).Difference(old.(*schema.Set))
+		oldUsernames, newUsernames := d.GetChange("usernames")
```

Comment:
I'm glad to see this change because the new variable names are more descriptive, although I find it interesting that the linter did this!

Reply:
The linter catches the case and shows the warning but does not give any suggestions of namings. One would have to manually rename it.

Reply:
Oh nice! ✨